avocado/core/output.py

```diff
@@ -643,6 +643,8 @@ def add_log_handler(logger, klass=logging.StreamHandler, stream=sys.stdout,
 
 
 def disable_log_handler(logger):
+    if not logger:
+        return
```

Comment:
@willianrampazzo I can narrow my problem in https://github.com/avocado-framework/avocado/issues/4797 to these two lines, if I revert those (keeping all other changes from this PR) everything works again.